lib/teleterm/daemon/daemon.go

```diff
@@ -384,13 +384,14 @@ func (s *Service) GetRequestableRoles(ctx context.Context, req *api.GetRequestab
 		return nil, trace.Wrap(err)
 	}
 
-	response, err := cluster.GetRequestableRoles(ctx)
+	response, err := cluster.GetRequestableRoles(ctx, req)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
 
 	return &api.GetRequestableRolesResponse{
-		Roles: response,
+		Roles:           response.RequestableRoles,
```

Comment:
I suppose it would help if I added an error check to GetRequestableRoles 🤦 
It panicked because if this errored (i.e. cert expired) we would just return things that didn't exist... or try to at least